src/client/Client.cc

```diff
@@ -9785,6 +9786,11 @@ int Client::statfs(const char *path, struct statvfs *stbuf,
 
   client_lock.Unlock();
   int rval = cond.wait();
+
+  if (root == NULL)
```

Comment:
AFAICT, `root` cannot be `NULL` here. Did you run into a problem which caused you to add this code? (Note: we access root unprotected in many other parts of the Client.)

Reply:
No, I haven't. I just wanted to be sure that the client won't crash while accessing the root.

Reply:
Let's remove the check then.

Reply:
I think I suggested this one because the root member doesn't get assigned until the client has received the inode from the MDS.  However, on second look I see that mount() doesn't return until root is set, so we should be good with an assert(root); here.